CatAnalyzer/test/run_ColorCoherence_b_cfg.py

```diff
@@ -0,0 +1,54 @@
+import FWCore.ParameterSet.Config as cms
+import os, sys
+process = cms.Process("ColorCoherenceAnalyzer")
+
+process.load("FWCore.MessageService.MessageLogger_cfi")
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(-1) )
+
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring()
+)
+in_txt = open(sys.argv[2])
+for f in in_txt:
+    process.source.fileNames.append(f)
```

Comment:
should work, but i think HY should remove this and use catGetDatasetInfo and create-batch